### DISCUSSION TITLE
Blitzwolf SHP5 / Gosund SP1 /Gosund SP112 disable blinking LED

### DISCUSSION THREAD

**Discussion by AleXSR700** [arendst/Tasmota#12852] [Q&A] [answered]
Hello everyone,

I am using some Gosund SP112 and Gosund SP1 devices with Tasmota and love the functionality!

The only thing I cannot figure out is something seemingly basic:
How do I disable the power button LED blinking in standby mode? When the device is turned off, the button blinks red. I would like it to simply be off when all power outlets are off.

Gosund SP112: Blitzwolf SP5 {"NAME":"SHP5","GPIO":[321,3072,320,3104,0,225,0,0,0,0,224,0,32,0],"FLAG":0,"BASE":18

Could someone help me out with this, please?

Thank you very much :)
Alex

**Comment by sfromis** [comment, marked as answer]
Something blinking is usually due to Tasmota failing to get a connection to Wifi or MQTT.
